hack/lib/etcd.sh

```diff
@@ -74,12 +74,16 @@ kube::etcd::start() {
 }
 
 kube::etcd::stop() {
-  kill "${ETCD_PID-}" >/dev/null 2>&1 || :
-  wait "${ETCD_PID-}" >/dev/null 2>&1 || :
+  if [[ -n "${ETCD_PID-}" ]]; then
+    kill "${ETCD_PID}" >/dev/null 2>&1 || :
```

Comment:
&>

Reply:
Ok, I added it to all places that were modified. Thanks!